aspnetcore/migration/inc/samples/remote-authentication/AspNetCore.cs

```diff
@@ -0,0 +1,19 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// <snippet_AddSystemWebAdapters>
+builder.Services.AddSystemWebAdapters()
+    .AddRemoteAppClient(options =>
+    {
+        options.RemoteAppUrl = new Uri(builder.Configuration["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"]);
```

Comment:
Different than the original one because I find using `new(builder.Configuration["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"])` harder to follow in the docs, and I like being explicit with using `new Uri()`.

I can revert this if that's preferred.

Reply:
```suggestion
        options.RemoteAppUrl = new Uri(builder.Configuration
        ["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"]);
```